docker/wholegraph/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM nvidia/cuda:11.3.1-devel-ubuntu20.04
+ENV DEBIAN_FRONTEND=noninteractive
+ENV HOME=/root
+
```

Comment:
Where do you install wholegraph and its dependancies?

Reply:
Pasted the wrong file. Fixed now.